documentation-site/pages/components/slider.mdx

```diff
@@ -36,27 +35,26 @@ Each slider has a primary control handle that cues user interaction.
   <Basic />
 </Example>
 
-<Example title="Slider as a Stateful Component" path="examples/slider/stateful.js">
+<Example
+  title="Slider as a Stateful Component"
+  path="examples/slider/stateful.js"
+>
   <Stateful />
 </Example>
 
-<Example title="Slider Range example" path="examples/slider/range.js">
+<Example title="Slider Range" path="examples/slider/range.js">
```

Comment:
can you add an example for how to achieve this use-case? `For example, if the defined range is [402, 8442] with 402 step increments, but you want to show "0.4km" or "0.25mi" as the label instead of "402"` This should be simple to do through overrides, but would be nice to help migration efforts

Reply:
Added this example:

<img width="769" alt="screen shot 2019-02-11 at 1 35 08 pm" src="https://user-images.githubusercontent.com/1387913/52595156-062dfc00-2e02-11e9-8a26-5e1d7a18cf11.png">

It also answers the multiple ticks use-case. Previously, it just displayed whatever user manually specified through the `range` prop. Since this prop doesn't exist anymore, you have to DIY through the `TickBar` override. Yep, that's a bit more work but we display the current value above the thumb by default so it doesn't seem so necessary (also no design for that).

Moreover, if we decided to solve this properly, `justify-content: space-between` doesn't cut it  since it's not responsive. There needs to be more logic to hide some elements when the viewport is too narrow. That's something that `d3` solves but I would say it's out of the scope here and we can introduce some dataviz like components separately in the future.